irods/test/data_obj_test.py

```diff
@@ -330,9 +324,9 @@ def test_put_get_parallel_autoswitch_A__235(self):
             options = { kw.DEST_RESC_NAME_KW:Root,
                         kw.RESC_NAME_KW:Root }
 
-            PUT_LOG = self.In_Memory_Stream(always_unicode = True)
```

Comment:
Is `In_Memory_Stream` a class or function that needs to be removed now?

Reply:
I've removed it.  It was part of the test suite and not really used/documented outside of that.